src/components/ProductCard/ProductCard.jsx

```diff
@@ -10,30 +10,40 @@ export function ProductCard(props) {
 
   const token = localStorage.getItem("Manazone.Token");
 
-  const { title, image, price, categoryName, stars, badge, id, _id } = props;
+  const { title, image, price, categoryName, stars, badge, id, _id, data_id } =
+    props;
   const { addToCart, cartData } = useCart();
   const { addToWishlist, wishlistData, removeFromWishlist } = useWishlist();
-
   const handleAddToCart = (props) => {
+    e.stopPropagation();
+
```

Comment:
you need to accept `e` as a parameter. This function is throwing an error

Reply:
Fixed the bug. Thanks for pointing it out !